script/Arcueid/Day 8/08_00_ARC08_8_2.txt

```diff
@@ -91,12 +95,15 @@ The mass of iron passes right above my head.
 [sha:515b1782f0ee27a05bbecec0903095e2cc41fb07]{
 -- Page 586, Offset 21259.
 -- 前のめりに地面に倒れ込んで、地上80センチのなぎ払いを躱す。
-Falling forward to the floor, I evade the sweeping attack 80 centimeters above ground.
+// He doesn't actually flop all the way down here, based on next line
+I pitch forward as though collapsing to the floor,
```

Comment:
I think the old TL is more right since he _is_ falling to the ground, he just stops himself before he hits it, right? Maybe something looser. Couple of options.
> Toppling back, I evade the sweep of her spear 80cm off the ground.
> Pushing my weight to the earth, I evade the sweep of her spear 80cm off the ground.

なぎ払うgives me the sense I get with the expression ["daisy cutter"](https://en.wikipedia.org/wiki/BLU-82) but I can't really put it into words.

Reply:
> I throw myself to the floor, evading her spear as it carves a swathe through the air 80cm above the ground.

?